src/elements/Typography.tsx

```diff
@@ -37,7 +37,7 @@ const Text = styled.Text.attrs<TextProps>({})`
   ${maxWidth};
 `
 
-interface TypographyProps {
+export interface TypographyProps {
```

Comment:
@zephraph - this is needed for `tsc --emit`, which will automatically emit type definition files for consumers.